src/test/java/picard/sam/FilterSamReadsTest.java

```diff
@@ -95,7 +95,7 @@ public void testPairedIntervalFilter(final String intervalFilename, final int ex
         // Build a sam file for testing
         final File inputSam = File.createTempFile("testSam", ".sam", TEST_DIR);
         inputSam.deleteOnExit();
-        final File sortedSamIdx = new File(TEST_DIR, inputSam.getName() + ".idx");
+        final File sortedSamIdx = new File(TEST_DIR, inputSam.getName() + ".bai");
```

Comment:
I can't comment on [the line](https://github.com/broadinstitute/picard/pull/843/files#diff-9c09d6174fd9f608424a44e8e09ac593R102) directly, but could it be because the test is using `makeBAMWriter` instead of `makeSAMWriter` below?